paper-calendar.html

```diff
@@ -289,6 +289,22 @@
           value: null
         },
         /**
+         *Array of selectable dates
+         * (whitelist)
+         */
```

Comment:
Add `Format must be String YYYY-MM-DD` here as well as on `invalidDates` so it's documented for people using this element stand-alone.